docs/quick-ingestion-guides/powerbi/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+---
+# PowerBI Ingestion Guide: Overview
+
+## What You Will Get Out of This Guide
+
+This guide will help you set up the PowerBI connector to begin ingesting metadata into DataHub.
+
+Upon completing this guide, you will have a recurring ingestion pipeline that will extract metadata from PowerBI and load it into DataHub. This will include to following PowerBI asset types:
+
+* Dashboards
+* Tiles 
+* Reports
+* Pages
+* Datasets
+* Lineage 
+
+
+
+*To learn more about setting these advanced values, check out the [PowerBI Ingestion Source](https://datahubproject.io/docs/generated/ingestion/sources/powerbi).*
+
+## Next Steps
+
+If that all sounds like what you're looking for, navigate to the [next page](setup.md), where we'll talk about prerequisites.
```

Comment:
`Continue to the setup guide, where we'll describe the prerequisites. `

Reply:
done